lib/http/server.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+const {
+  Promise,
+  RegExpPrototypeExec,
+  StringPrototypeEndsWith,
+  StringPrototypeIndexOf,
+  StringPrototypeLastIndexOf,
+  StringPrototypeReplace,
+  StringPrototypeSlice,
+  StringPrototypeStartsWith,
+  StringPrototypeToLocaleLowerCase,
+} = primordials;
+
+const { cwd, nextTick } = process;
+const console = require('internal/console/global');
+const { createReadStream } = require('fs');
+const { Server } = require('_http_server');
+const { URL, isURLInstance, pathToFileURL } = require('internal/url');
+const { sep } = require('path');
+const { emitExperimentalWarning } = require('internal/util');
+
+emitExperimentalWarning('http/server');
+
+if (module.isPreloading) {
```

Comment:
This is dangerous. If someone has their own preload module and happens to `require('http/server')` within it, this part will be run while they may not expect it.